src/ui/pages/SwapForm/fieldsets/ReceiveTokenField/MarketAssetSelect/MarketAssetSelect.tsx

```diff
@@ -42,16 +42,24 @@ export function MarketAssetSelect({
       ),
     [addressPositions]
   );
+
+  const { data: popularAssetCodes, isLoading: popularAssetCodesAreLoading } =
+    useQuery({
+      queryKey: ['getPopularTokens', chain],
+      queryFn: () => getPopularTokens(chain),
+      suspense: false,
+      retry: false,
+      staleTime: Infinity,
```

Comment:
I guess they rarely change... but should we **never** refresh the popular tokens? Maybe we could set it to smth in range of `1 * 60 * 1000 < staleTime < 5 * 60 * 1000`... idk

Reply:
[ah, alright](https://github.com/zeriontech/zerion-wallet-extension/pull/711#pullrequestreview-2640986518) :)